src/main/java/com/dougnoel/sentinel/elements/Element.java

```diff
@@ -952,4 +952,25 @@ public Point getLocation(){
 		return element().getLocation();
 	}
 
+	/**
+	 * Returns the stored XPath for this element.
+	 * Returns null if no XPath exists in the page object.
+	 *
+	 * @return the xPath for the element listed in the page object
+	 */
+	public By xPathToBy() {
```

Comment:
See if we can make these private/protected

Reply:
Private and Protected created an error  on this By enterTextLocator = getElement(elementName).xPathToBy(ordinalRow); It was suggested to make it public